modules/nw-ovn-kuberentes-limitations.adoc

```diff
@@ -25,3 +25,15 @@ F1006 16:09:50.985939   60651 ovnkube.go:130] multiple gateway interfaces detect
 ----
 +
 The only resolution is to reconfigure the host networking so that both IP families use the same network interface for the default gateway.
+
+* For clusters configured for dual-stack networking, both the IPv4 and IPv6 routing tables must contain the default gateway.
+If this requirement is not met, pods on the host in the `ovnkube-node` daemon set enter the `CrashLoopBackOff` state.
+If you display a pod with a command like `oc get pod -n openshift-ovn-kubernetes -l app=ovnkube-node -o yaml`, the `status` field contains more than one message about the default gateway, as shown in the following output:
```

Comment:
I am guessing that you copied the previous bullet, where this was approved. Can I ask you to change the _like_ in both bullets to _such as_? The IBM Style Guide says not to use _like_ in this way. Thank you!